src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,135 +32,32 @@ contract DssSpellCollateralAction {
     //
 
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
-    // uint256 constant MILLION    = 10 ** 6;
-    // uint256 constant BILLION    = 10 ** 9;
+    uint256 constant MILLION = 10 ** 6;
 
-    // --- DEPLOYED COLLATERAL ADDRESSES ---
-    // address constant XXX                  = 0x0000000000000000000000000000000000000000;
-    // address constant PIP_XXX              = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_JOIN_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_CALC_XXX_A  = 0x0000000000000000000000000000000000000000;
-
-    // --- Offboarding: Current Liquidation Ratio ---
-    // uint256 constant CURRENT_XXX_A_MAT              =  XYZ * RAY / 100;
-
-    // --- Offboarding: Target Liquidation Ratio ---
-    // uint256 constant TARGET_XXX_A_MAT               =  XYZ * RAY / 100;
+    bytes32 constant RWA007_A                 = "RWA007-A";
+    uint256 constant RWA007_A_AUTOLINE_AMOUNT = 250 * MILLION;
+    uint256 constant RWA007_A_AUTOLINE_GAP    =  50 * MILLION;
+    uint256 constant RWA007_A_AUTOLINE_TTL    = 1 weeks; 
```

Comment:
Config uses 604_800 (no unit keyword) with 'weeks', proposing we match this.

```suggestion
    uint256 constant RWA007_A_AUTOLINE_TTL    = 604_800; // 1 week 
```

Reply:
I would prefer to use two different measures since config is a test and therefore provides a check that 1 weeks is what we expect and that 604_800 seconds is one week

Reply:
So, we test this in terms of seconds later in the test.  It think I prefer this notation for now, so long as we are testing against the seconds value in the integration tests (which we are).

Reply:
Makes sense, no issue here then.